configs/platforms/amazon-2023-x86_64.rb

```diff
@@ -0,0 +1,38 @@
+platform "amazon-2023-x86_64" do |plat|
+  plat.servicedir "/usr/lib/systemd/system"
+  plat.defaultdir "/etc/sysconfig"
+  plat.servicetype "systemd"
+
+  packages = %w(
```

Comment:
Why do we inherit from default for aarch64 but not x86_64? It looks like we could remove a lot of redundant packages if we inherited the amazon-2023-x86_64 Vanagon defaults.

Reply:
I had copied the el-9-x86_64 & el-9-aarch64 platform file for amazon linux 2023. I have now rectified the files for amazon-linux-2023 to have 'inherit_from_default' for both the architectures.

I have also removed the redundant packages and kept only the required ones.